aten/src/ATen/native/native_functions.yaml

```diff
@@ -2554,6 +2554,13 @@
     CUDA: layer_norm_cuda
     CompositeImplicitAutograd: math_native_layer_norm
 
+- func: attn(Tensor q, Tensor k, Tensor v) -> (Tensor, Tensor)
```

Comment:
Another observation. 
In the previous commit, I wrote " -> (Tensor o, Tensor a)". It appears that namedtuple results requires additional registration, e.g. joining all_operators_with_namedtuple_return... 

Why namedtuple results has this additional check? How is named results different that requires such special enforcement?

Reply:
It is different because there needs to be some extra code that creates that named tuple class and exposes it to python properly.

Reply:
Got it. Thanks!